buddyboss/buddyboss-platform
# PROD-748 Forum reply notification issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2879  .

### How to test the changes in this Pull Request:

1. Create a forum reply to another person's reply
2. Login to the person where you create the reply
3. See the notification it shows the correct name.

### Proof Screenshots or Video

https://www.loom.com/share/c4a3da6dff414ce9997ccc0485582f45

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
